sunpy/__init__.py

```diff
@@ -15,11 +15,15 @@
 from sunpy.util import system_info
 from sunpy.util.config import load_config, print_config
 from sunpy.util.logger import _init_log
-from .version import __version__
 
 # Enforce Python version check during package import.
 __minimum_python_version__ = "3.6"
 
+try:
+    from .version import __version__
+except ImportError:
+    print("version.py not found, please reinstall sunpy.")
+    __version__ = "0.0.0"
```

Comment:
Not sure if we want to do this but I clean my git repo all the time and reinstalling sunpy is a pain.

Reply:
```suggestion
    __version__ = "unknown"
```